python-spec/src/somacore/scene.py

```diff
@@ -90,6 +91,35 @@ class Scene(
     Lifecycle: experimental
     """
 
+    @classmethod
+    @abc.abstractmethod
+    def create(
+        cls,
+        uri: str,
+        *,
+        coordinate_space: Optional[
+            Union[Sequence[str], coordinates.CoordinateSpace]
+        ] = None,
+        platform_config: Optional[options.PlatformConfig] = None,
+        context: Optional[Any] = None,
+    ) -> Self:
+        """Creates a new collection of this type at the given URI.
+
+        Args:
+            uri: The URI where the collection will be created.
+            coordinate_space: Optional coordinate space or the axis names for the
+                coordinate space the scene is defined on. Defaults to ``None``.
```

Comment:
Say what happens if `None` -- does it mean the resulting `MultiscaleImage` will not have any coordinate space? Or that we will use a default? And if the latter, please say what we will use.